ui/component/comment/view.jsx

```diff
@@ -149,6 +167,13 @@ function Comment(props: Props) {
                   <MenuItem className="comment__menu-option" onSelect={handleDeleteComment}>
                     {__('Delete')}
                   </MenuItem>
+                  {parentId === null ? (
```

Comment:
You'll need to move the `commentIsMine` line down to inside the menu. Right now it looks like you would only be able to reply to your own comments.

Reply:
Also instead of the ternary you can just do `parentId === null && <MenuItem...`

Reply:
Something breaks in the MenuList when I don't do ternary. Same thing with `<React.Fragment>` on the remaining commentIsMine elements (Edit and Delete). So if I use ternary with all of them it works, like this:
```js
<MenuList className="comment__menu-list">
  {isCommentMine() ? (
	  <MenuItem className="comment__menu-option" onSelect={handleDeleteComment}>
		{__('Delete')}
	  </MenuItem>
  ) : ('')}
  {isCommentMine() ? (
	<MenuItem className="comment__menu-option" onSelect={handleSetEditing}>
	  {__('Edit')}
	</MenuItem>
  ) : ('')}
  {isParent() ? (
	<MenuItem className="comment__menu-option" onSelect={handleReply}>
	  {__('Reply')}
	</MenuItem>
  ) : ('')}
</MenuList>

```

If I do this:
```js
<MenuList className="comment__menu-list">
  {isCommentMine() ? (
    <React.Fragment>
	<MenuItem className="comment__menu-option" onSelect={handleDeleteComment}>
	  {__('Delete')}
	</MenuItem>
	<MenuItem className="comment__menu-option" onSelect={handleSetEditing}>
	  {__('Edit')}
	</MenuItem>
    </React.Fragment>
  ) : ('')}
  {isParent() ? (
	<MenuItem className="comment__menu-option" onSelect={handleReply}>
	  {__('Reply')}
	</MenuItem>
  ) : ('')}
</MenuList>
```
It doesn't work.

I'll put a new commit for what works currently

Reply:
Ah, I remember the menu list being weird about those. Sorry about that, the ternary should be fine.